librz/core/canalysis.c

```diff
@@ -707,16 +589,116 @@ RZ_API void rz_core_analysis_autoname_all_golang_fcns(RzCore *core) {
 	}
 }
 
-/* suggest a name for the function at the address 'addr'.
- * If dump is true, every strings associated with the function is printed */
-RZ_API char *rz_core_analysis_fcn_autoname(RzCore *core, ut64 addr, int dump, int mode) {
-	RzAnalysisFunction *fcn = rz_analysis_get_fcn_in(core->analysis, addr, 0);
-	if (fcn) {
-		return analysis_fcn_autoname(core, fcn, dump, mode);
+static bool blacklisted_word(const char *name) {
```

Comment:
I would suggest to use that refactoring to move that in a static file that way, it will be easier for people to contribute